api/src/main/java/api/Answer.java

```diff
@@ -24,4 +27,19 @@ public void setAccepted(boolean accepted) {
         this.accepted = accepted;
     }
 
+    public long getQuestionId() {
+        return questionId;
+    }
+
+    public void setQuestionId(long questionId) {
+        this.questionId = questionId;
+    }
+
+    public Question getQuestion() {
```

Comment:
This is never populated in a dao right? and not used by the ui? Maybe we can add a jsonIgnore to this property. I dont see the use case of exposing it to the front-end. I think that the frontend will do a getQuestion(id) and getAnswer(aId) anyway?

Its used internally, but not really something that we would want to expose. Im thinking about if its possible to use default visibility on this method if possible. Otherwise it will be exposed as a part of the java api as well

Reply:
It's populated in getAnswerById but not used in the ui. I'll try to find a way not to expose it.